code-of-conduct.md

```diff
@@ -0,0 +1,79 @@
+
+Exhibitors should not use sexualized images, activities, or other material in their booths/areas and must refrain from the use of sexualized clothing, uniforms, costumes, or otherwise creating a sexualized environment. Speakers should not use sexual language, images, or any language or images that would constitute harassment as defined above in their talks.
+
+Individuals who participate (or plan to participate) in Orchestructure events should conduct themselves at all times in a manner that comports with both the letter and spirit of this policy prohibiting harassment and abusive behavior, whether before, during or after the event. This includes statements made in social media postings, online publications, text messages, and all other forms of electronic communication.
+
+#### Consequences of Unacceptable Behavior
+
+If a participant engages in harassing behavior, the event organizers may take any action they deem appropriate depending on the circumstances, ranging from issuance of a warning to the offending individual to expulsion from the conference with no refund. Orchestructure organizers reserve the right to exclude any participant found to be engaging in harassing behavior from participating in any further Orchestructure events, trainings or other activities.
+
+If a participant (or individual wishing to participate in a Orchestructure event), through postings on social media or other online publications or another form of electronic communication, engages in conduct that violates this policy, whether before, during or after a Orchestructure event, the Orchestructure organizers may take appropriate corrective action, which could include imposing a temporary or permanent ban on an individual’s participation in future Orchestructure events.
+
+#### What to do if you Witness or are Subject to Unacceptable Behavior
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of event staff immediately. Event staff introduce themselves at the beginning of every event, can be identified by t-shirts/staff badges, and are always available at every Orchestructure event. You are also encouraged to contact the Orchestructure organizers at orchestructure.meetup (at) gmail (dot) com.
```

Comment:
```suggestion
If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of [Orchestructure Organizers](https://www.meetup.com/orchestructure/members/?op=leaders). The Organizers will introduce themselves at the beginning of every event, with at least one on-site at all times for its duration. The Orchestructure Organizers may also be contacted via email at orchestructure.meetup (at) gmail (dot) com.
```

Since we don't really have event staff, deferred to just organizers.